models/TvShow.js

```diff
@@ -1,9 +1,9 @@
 const {Model,DataTypes}=require('sequelize');
 const sequelize=require('../config/connection');
 
-class Show extends Model{}
+class TvShow extends Model{}
 
-Show.init(
+TvShow.init(
```

Comment:
I think we ought to associate the user with the tv show that is added and then use that to filter the shows for the dashboard, or else it's going to display every tv show under the sun in the user's personal dashboard

Reply:
well the requirement says that it's colored if the subscription is active. we can filter by active subscriptions but then we wouldn't need to color code it.

Reply:
![image](https://user-images.githubusercontent.com/103607215/185260098-5b886d73-5d2c-483e-abea-d8bd8da5d110.png)

Reply:
Not saying there's anything wrong with your current implementation; it all looks correct -- but the user dashboard should display things specific to the user.